test/controllers/still.required.controllers.spec.unit.ts

```diff
@@ -34,4 +34,55 @@ describe("still required validation tests", () => {
     expect(response.text).toContain("Tell us if THE GIRLS DAY SCHOOL TRUST is required");
     expect(response.text).toContain(COMPANY_REQUIRED_NOT_SELECTED);
   });
+
+  it("should update session when yes is selected", async () => {
+    const response = await request(app)
+        .post("/promise-to-file/company/00006400/still-required")
+        .set("Referer", "/")
+        .set("Cookie", [`${COOKIE_NAME}=123`])
+        .send({stillRequired: "yes"});
+
+    expect(response.status).toEqual(302);
+    expect(mockUpdatePromiseToFileSessionValue).
+    toHaveBeenCalledWith(buildDummySession("123", "00006400"), IS_STILL_REQUIRED, true);
+  });
+
+  it("should update session when yEs is selected", async () => {
+    const response = await request(app)
+        .post("/promise-to-file/company/00006400/still-required")
+        .set("Referer", "/")
+        .set("Cookie", [`${COOKIE_NAME}=123`])
+        .send({stillRequired: "yEs"});
+
+    expect(response.status).toEqual(302);
+    expect(mockUpdatePromiseToFileSessionValue).
+    toHaveBeenCalledWith(buildDummySession("123", "00006400"), IS_STILL_REQUIRED, true);
+  });
+
+  it("should update session when no is selected", async () => {
+
+    const response = await request(app)
+        .post("/promise-to-file/company/00006400/still-required")
+        .set("Referer", "/")
+        .set("Cookie", [`${COOKIE_NAME}=123`])
+        .send({stillRequired: "no"});
+
+    expect(response.status).toEqual(302);
+    expect(mockUpdatePromiseToFileSessionValue).
+    toHaveBeenCalledWith(buildDummySession("123", "00006400"), IS_STILL_REQUIRED, false);
+  });
+
+  it("should update session when nO is selected", async () => {
+
+    const response = await request(app)
+        .post("/promise-to-file/company/00006400/still-required")
+        .set("Referer", "/")
+        .set("Cookie", [`${COOKIE_NAME}=123`])
+        .send({stillRequired: "nO"});
```

Comment:
Is n0 being interpreted as false here? Should we be doing that? I can understand making it case insensitive, but I personally wouldn't accept n0 as valid input here

Reply:
It is just to check that the to upper case is working properly it is a capital 'O' not a zero.